readme.md

```diff
@@ -64,7 +64,7 @@ The goal of this rewrite is two-folded:
 
 ## Installation
 
-    npm install tabtab
+    npm install @pnpm/tabtab
```

Comment:
**Nit:** Since this is a @pnpm project, it should suggest [pnpm](https://pnpm.js.org) as the recommended installer:
````suggestion
```
pnpm add @pnpm/tabtab
```
````